lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -310,6 +313,8 @@ namespace Js
 
     void ScriptContext::InitializeAllocations()
     {
+        this->jitFuncRangeCache = HeapNew(JITPageAddrToFuncRangeCache);
```

Comment:
>this->jitFuncRangeCache = HeapNew(JITPageAddrToFuncRangeCache); [](start = 8, length = 63)

Should be in ScriptContext::InitializePreGlobal where we create nativeCodeGen and under ENABLE_NATIVE_CODEGEN